web/Views/Landlord/Dashboard.cshtml

```diff
@@ -7,62 +7,90 @@
 
 <div>
     <div class="row">
-        <div class="col-1">
-            @* TODO BNH-99: Replace icon with profile picture *@
-            <img alt="profile picture" src="https://www.softwire.com/wp-content/uploads/2019/07/Gareth-J-Final-Head-Shot.jpg" width="100" height="100" class="rounded-circle img-thumbnail img-fluid"/>
-        </div>
-        <div class="col-7">
+        <div class="col">
             <div class="row">
-                <div class="col-7">
-                    <h6>Hello</h6>
-                    <h2 class="font-blue">@Model.FirstName</h2>
-                    <h6>Start adding your properties to the portal, to get matches</h6>
-                    <div>
-                        <a class="btn btn-primary mt-3" href='@Url.Action("AddNewProperty_Begin", "Property", null)'>Add Properties</a>
-                    </div>
-                </div>
-            </div>
-            <div class="row mt-4">
-                <div class="col-5">
-                    <div class="row">
-                        <div class="col-3">
-                            @if (Model.CharterApproved)
-                            {
-                                <h1>
-                                    <i class="bi bi-check"></i>
-                                </h1>
-                            }
-                            else
-                            {
-                                <h1>
-                                    <i class="bi bi-x"></i>
-                                </h1>
-                            }
-                        </div>
-                        <div class="col-5">
-                            Charter Approval State
-                        </div>
-                    </div>
+                <div class="col-sm-auto">
+                    @* TODO BNH-99: Replace icon with profile picture *@
+                    <img alt="profile picture" src="https://www.softwire.com/wp-content/uploads/2019/07/Gareth-J-Final-Head-Shot.jpg" width="100" height="100" class="rounded-circle img-thumbnail img-fluid"/>
```

Comment:
I think this may have been updated, if not it should bbe updated with the new image